packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -392,8 +392,7 @@ export async function generateCrudInput(
                 const decorator = definedDecorators.find((decorator) => {
                     return (
                         // ignore casing since class validator is inconsistent with casing
-                        decorator.getName().toUpperCase() === constraint.name.toUpperCase() ||
-                        // some class validator decorators have a prefix "Is" but not in the constraint name
+                        decorator.getName().toUpperCase() === constraint.name.toUpperCase() || // some class validator decorators have a prefix "Is" but not in the constraint name
```

Comment:
This change seems unrelated to me

Reply:
that happend because of the applied MUI Codemod, i dont know how exactly, i can try to revert this.